frontend/www/js/omegaup/components/contest/Report.vue

```diff
@@ -26,75 +26,71 @@
           <h3>{{ T.wordsPoints }}: {{ item.points }}</h3>
           <div v-if="item.run_details">
             <template v-for="group in item.run_details.details.groups">
-              <table
-                v-if="
-                  item.run_details &&
-                  (((item || {}).run_details || {}).details || {}).groups
-                "
-                :key="`${contestantData.username}_${item.alias}_${group.group}_case`"
-                class="table table-stripped table-responsive"
-              >
-                <thead>
-                  <tr class="text-center">
-                    <th scope="col">{{ T.wordsCase }}</th>
-                    <th scope="col">{{ T.wordsTimeInSeconds }}</th>
-                    <th scope="col">{{ T.wordsWallTimeInSeconds }}</th>
-                    <th scope="col">{{ T.wordsMemoryInMebibytes }}</th>
-                    <th scope="col">{{ T.wordsStatus }}</th>
-                    <th scope="col">{{ T.rankScore }}</th>
-                    <th scope="col">{{ T.wordsDifference }}</th>
-                  </tr>
-                </thead>
-                <tbody>
-                  <tr
-                    v-for="groupCase in group.cases"
-                    :key="`${contestantData.username}_${item.alias}_${group.group}_${groupCase.name}_case`"
-                  >
-                    <th scope="row">{{ group.group }}.{{ groupCase.name }}</th>
-                    <td class="text-right">
-                      {{ groupCase.meta.time.toFixed(3) }}
-                    </td>
-                    <td class="text-right">
-                      {{ groupCase.meta.wall_time.toFixed(3) }}
-                    </td>
-                    <td class="text-right">
-                      {{ groupCase.meta.memory.toFixed(2) }}
-                    </td>
-                    <td class="text-center">{{ groupCase.verdict }}</td>
-                    <td class="text-center">{{ groupCase.score }}</td>
-                    <td>
-                      <template v-if="groupCase.out_diff">
-                        {{ groupCase.out_diff }}
-                      </template>
-                    </td>
-                  </tr>
-                </tbody>
-              </table>
+              <template v-if="item.run_details.details.groups">
```

Comment:
```suggestion
              <template v-if="item.run_details?.details?.groups">
```

acá sí se necesita el optional chaining porque todos esos pueden ser nulos.

Reply:
Este `v-if` se podría eliminar si el `v-for` de abajo regresa un array vacío. Así que ya no es necesario